test/unit/views/ChoiceViewSpec.scala

```diff
@@ -179,8 +179,8 @@ class ChoiceViewSpec extends UnitViewSpec with ChoiceMessages with CommonMessage
   }
 
   private def ensureRadioIsChecked(view: Document, elementId: String): Unit = {
-    val option = view.getElementById(elementId)
-    option.attr("checked") mustBe "checked"
+    val option = view.getElementById(elementId).getElementsByAttribute("checked")
+    option.size() mustBe 1
```

Comment:
Now we are not sure that this is expected element

Reply:
We are because we use the `elementId`